components/dashicon/index.js

```diff
@@ -892,7 +892,9 @@ export default class Dashicon extends wp.element.Component {
 				height={ size }
 				viewBox="0 0 20 20"
 			>
-				<path d={ path } />
+				<path d={ path }>
+					{ title && <title>{ title }</title> }
```

Comment:
Hmm, I don't know enough about whether this is something we actually want. Can you explain context for adding this title?

cc @afercia

Reply:
I added it so that curious users can hover their cursor over the icon and get a little explanatory tooltip:

<img width="326" alt="screen shot 2018-03-22 at 10 03 22" src="https://user-images.githubusercontent.com/612155/37742135-60d92f4e-2db8-11e8-9712-91261e92ed7a.png">

Reply:
Should we use the `<Tooltip />` component instead for consistency?

Reply:
That's a good idea. Does `<Tooltip />` work for non-interactive elements?

Reply:
👌 done.